contracts/staking/test/actors/staker_actor.ts

```diff
@@ -52,95 +51,186 @@ export class StakerActor extends BaseActor {
+        const stakerBalances: StakeBalances = {
             zrxBalance: await this._stakingWrapper.getZrxTokenBalanceAsync(this._owner),
             stakeBalance: await this._stakingWrapper.getTotalStakeAsync(this._owner),
             stakeBalanceInVault: await this._stakingWrapper.getZrxVaultBalanceAsync(this._owner),
             withdrawableStakeBalance: await this._stakingWrapper.getWithdrawableStakeAsync(this._owner),
-            activatableStakeBalance: await this._stakingWrapper.getActivatableStakeAsync(this._owner),
-            activatedStakeBalance: await this._stakingWrapper.getActivatedStakeAsync(this._owner),
-            timeLockedStakeBalance: await this._stakingWrapper.getTimeLockedStakeAsync(this._owner),
-            deactivatedStakeBalance: await this._stakingWrapper.getDeactivatedStakeAsync(this._owner),
+            activeStakeBalance: await this._stakingWrapper.getActiveStakeAsync(this._owner),
+            inactiveStakeBalance: await this._stakingWrapper.getInactiveStakeAsync(this._owner),
+            delegatedStakeBalance: await this._stakingWrapper.getStakeDelegatedByOwnerAsync(this._owner),
+            delegatedStakeByPool: {},
+            totalDelegatedStakeByPool: {},
         };
+        // lookup for each pool
+        for (const poolId of this._poolIds) {
+            const delegatedStakeBalanceByPool = await this._stakingWrapper.getStakeDelegatedToPoolByOwnerAsync(
+                poolId,
+                this._owner,
+            );
+            const totalDelegatedStakeBalanceByPool = await this._stakingWrapper.getTotalStakeDelegatedToPoolAsync(
+                poolId,
+            );
+            stakerBalances.delegatedStakeByPool[poolId] = delegatedStakeBalanceByPool;
+            stakerBalances.totalDelegatedStakeByPool[poolId] = totalDelegatedStakeBalanceByPool;
+        }
         return stakerBalances;
     }
-    public async assertBalancesAsync(expectedBalances: StakerBalances): Promise<void> {
+    public async assertBalancesAsync(expectedBalances: StakeBalances): Promise<void> {
         const balances = await this.getBalancesAsync();
         expect(balances.zrxBalance, 'zrx balance').to.be.bignumber.equal(expectedBalances.zrxBalance);
-        expect(balances.stakeBalance, 'stake balance').to.be.bignumber.equal(expectedBalances.stakeBalance);
         expect(balances.stakeBalanceInVault, 'stake balance, recorded in vault').to.be.bignumber.equal(
             expectedBalances.stakeBalanceInVault,
         );
         expect(balances.withdrawableStakeBalance, 'withdrawable stake balance').to.be.bignumber.equal(
             expectedBalances.withdrawableStakeBalance,
         );
-        expect(balances.activatableStakeBalance, 'activatable stake balance').to.be.bignumber.equal(
-            expectedBalances.activatableStakeBalance,
+        expect(balances.activeStakeBalance.current, 'active stake balance (current)').to.be.bignumber.equal(
```

Comment:
+1 for the named expect. This is super nice to read, and I'm sure it's nice to debug.